src/utils/response.ts

```diff
@@ -148,6 +148,16 @@ export function appendResponseHeader(
 
 export const appendHeader = appendResponseHeader;
 
+export function removeResponseHeaders(event: H3Event): void {
```

Comment:
I think this should support a `string[]` no? What is usecase of removing all headers?

Reply:
I still don't see the reason. Why they should come from many places and need to be overridden like this? Do you have a real-world example? (If there is, we might also call it `clearResponseHeaders` for clarity)

Reply:
Supporting `string[]` for `removeResponseHeader` would be a bonus 💯

Reply:
For a Nuxt use case, you could have middlewares/modules that adds headers, and in a specific handler, you'd want to override all of them ?